modules/manage/pages/audit-logging.adoc

```diff
@@ -5,7 +5,7 @@ Many scenarios for streaming data include the need for fine-grained auditing of
 
 Redpanda's auditing capabilities support auditing both administrative and operational interactions with topics and with users. We comply with the open cybersecurity schema framework, providing a predictable and extensible solution which works seamlessly with industry standard tools.
 
-NOTE: All audit logging is configured at the Cluster level. Redpanda supports excluding specific topics from audit to help reduce noise in the log. Audit logging is disabled by default.
```

Comment:
This actually will be true. I know I previously said it was P2 but Mike went ahead and created the PR for this Friday